content/blog/open-sourcing-workshops-on-demand-part4-managing-the-backend.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Open Sourcing Workshops-on-Demand part 4: Manage the Backend"
```

Comment:
If you can, please modify the capitalization of the title to:
Open sourcing Workshops-on-Demand Part 4: Manage the backend